js/background.js

```diff
@@ -832,15 +832,70 @@
         ourNumber,
         'private'
       );
+
+      const readFile = attachment =>
+        new Promise((resolve, reject) => {
+          const FR = new FileReader();
```

Comment:
Sorry for the nit, but `FR` doesn't look like a great style match with the rest of the code.
`fileReader` would be more consistent?

Reply:
This was copy pasted from `file_input_view.js`, where they already used `FR`, so I left it unchanged *for consistency* ;) I changed to `fileReader` as you suggested.